src/components/View/Farmacia/FarmaciaSections/index.jsx

```diff
@@ -0,0 +1,13 @@
+import FarmaciaContacts from './FarmaciaContacts';
```

Comment:
questo file non serve, manteniamo la stessa struttura dei file fatta con le viste degli altri ct. 
Questi export vanno spostati in io-sanita-theme/components/View/Farmacia/index.js

e dentro index js vanno esportati anche loro con loadable cosi vengono messi nel chunk corretto